contracts/src/v0.8/automation/dev/chains/ArbitrumModule.sol

```diff
@@ -16,8 +16,7 @@ contract ArbitrumModule is ChainModuleBase {
   address private constant ARB_GAS_ADDR = 0x000000000000000000000000000000000000006C;
   ArbGasInfo private constant ARB_GAS = ArbGasInfo(ARB_GAS_ADDR);
 
-  uint256 private constant FIXED_GAS_OVERHEAD = 20000;
-  uint256 private constant PER_CALLDATA_BYTE_GAS_OVERHEAD = 20;
+  uint256 private constant FIXED_GAS_OVERHEAD = 5000;
```

Comment:
Is this finalized?

Reply:
yes this includes the gas usage for 3 calls combined:
`ARB_SYS.arbBlockNumber();`
`ARB_SYS.arbBlockHash(n);` and
`ARB_GAS.getCurrentTxL1GasFees();`
each takes about 1200 gas based on g1 - g2 formula.
and provide a little buffer.